### ISSUE TITLE
FEM, GUI, post processing, graphical output of line mesh results

### ISSUE THREAD

**Issue by FreeCAD-Bug-Importer** [FreeCAD/FreeCAD#5686]
Issue imported from https://tracker.freecad.org/view.php?id=2950

* **Reporter:** berndhahnebach
* **Date submitted:** 3/6/2017
* **FreeCAD version:** 0.17
* **Category:** Feature
* **Status:** confirmed
* **Tags:** 

# Original report text

See forum topics:

graphical:
https://forum.freecadweb.org/viewtopic.php?f=18&t=21013
https://forum.freecadweb.org/viewtopic.php?f=18&t=20076&start=40#p162519

section forces
see issue https://www.freecadweb.org/tracker/view.php?id=3126

# Other bug information

* **Priority:** normal
* **Severity:** feature
* **Category:** Feature
* **Updated:** 2/6/2021

**Comment by luzpaz** [comment]
@ickby if you have a moment, can you weigh-in on this FEM post-processing feature request ?

**Comment by ickby** [comment]
I don't think there is a ready to use vtk filter for this. However, it should be rather simple to implement something by hand, especially now with vtk python integration. However, not sure about the utility, are there many people using this or needing this?

**Comment by FEA-eng** [comment]
> However, not sure about the utility, are there many people using this or needing this?

Yes, it’s quite commonly requested. Some recent forum threads (older ones are listed in the description of the issue):
https://forum.freecad.org/viewtopic.php?t=88156
https://forum.freecad.org/viewtopic.php?t=100755

Generally speaking, it’s surprisingly hard to find decent open-source software for 3D beam/frame calculations. FreeCAD FEM has general-purpose solvers and beam elements (the CalculiX ones are not true 1D elements though as they get internally expanded to solids). However, this kind of postprocessing would greatly increase the usability of FreeCAD FEM for such structural calculations.

**Comment by luzpaz** [comment]
> Yes, it’s quite commonly requested

JFYI @ickby
